### DISCUSSION TITLE
タイポ？

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#200] [要望・アイデア・不具合]
[期間限定の記事投稿キャンペーン「Qiita Engineer Festa 2022」を6〜7月に開催決定！](https://blog.qiita.com/qiita-engineer-festa-2022-sponsor/)より、タイポと思われる箇所を発見したので報告です。

以下リンクより見出しの部分が「「Qiita Qiita Engineer Festa 2022」とは？」となっていますが、Qiitaが1個多いと思いました。ご確認お願いします。

https://blog.qiita.com/qiita-engineer-festa-2022-sponsor/#qiita-qiita-engineer-festa-2022%E3%81%A8%E3%81%AF

**Comment by kskwtnk** [comment]
@Brutus03 
ご報告ありがとうございます！修正いたしました！！ 🙏
